Models/DTOS/HTTP/Requests/GetUserPostsRequestDTO.cs

```diff
@@ -0,0 +1,7 @@
+namespace Tokengram.Models.DTOS.HTTP.Requests
+{
+    public class GetUserPostsRequestDTO : PaginationAbstractRequestDTO
```

Comment:
Discussed issue in DMs

Reply:
I kept the logic (for future use maybe), just made the parameter to be optional. I have also added posts ordering based on the time of acquisition. We can merge the PR after the time of acquisition is added to the indexer